duet/views/google_uac_android_activation.view.lkml

```diff
@@ -18,7 +18,7 @@ view: google_uac_android_activation {
          SELECT
            first_seen_date as date,
            REGEXP_REPLACE(adjust_campaign, '([\\w]+).*', '\\1') as campaign,
-           adjust_adgroup as ad_group,
+           REGEXP_REPLACE(REGEXP_REPLACE(adjust_adgroup, '(.*) \\([\\d]+\\)', '\\1'), ' - benchmark$', '') as ad_group,
```

Comment:
Why is it `'(.*) \\([\\d]+\\)'` and not `'(.*) ([\\d]+)'`? I'm not clear on what those first and last slashes do.

Might be helpful to also add a comment with an example?

Reply:
Still think it would be helpful to throw an example in the comment:

`-- e.g., 'my_group (123)' -> 'my_group'

Reply:
Good point, added!